app/src/main/java/com/example/tasky/agenda/presentation/components/AgendaItem.kt

```diff
@@ -181,25 +181,29 @@ fun AgendaItem(
                 }
                 Column {
                     Spacer(modifier = Modifier.size(2.dp))
+                    var showMenu by rememberSaveable {
+                        mutableStateOf(false)
+                    }
```

Comment:
I´d use remember instead of rememberSaveable

The only advantage "saveable" has, is that it survives process death
For a dialog, you wouldn´t expect it to survive process death